corehq/apps/reports/templates/reports/filters/form_app_module_drilldown.html

```diff
@@ -75,23 +75,40 @@
 {% endif %}
 {% endblock %}
 
-{% block filter_js %} {{ block.super }}
```

Comment:
@benrudolph this is where the `drilldownOptionFilter` instantiation is "deleted" - it's defined in the super block of `{% block filter_js %}` (and there's nothing else defined in that block).  By removing the `super` call, I'm replacing that block with this one, and it won't affect any of the custom reports that use the drilldown filter.  This file, `form_app_module_drilldown.html` is only in the corresponding filter class, so I think it's safe to change this.